other-identifiers/src/main/java/org/reactome/release/otherIdentifiers/PostStepChecks.java

```diff
@@ -38,31 +46,37 @@ public static void compareOtherIdentifierCounts(MySQLAdaptor dba, MySQLAdaptor d
      * @param dba MySQLAdaptor
      * @param speciesInstance GKInstance pertaining to a specific species
      * @return The other identifier count, as an int.
-     * @throws Exception
+     * @throws Exception can be caused by errors interacting with the database using the db adaptor.
```

Comment:
Could this be a little bit more specific to say the error would be in attempting to retrieve ReferenceGeneProduct instances for the passed species?